src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -316,5 +319,6 @@ def connect_to_fluent(
         watchdog.launch(os.getpid(), port, password, ip)
 
     return new_session(
-        fluent_connection=fluent_connection, remote_file_handler=RemoteFileHandler()
+        fluent_connection=fluent_connection,
+        remote_file_handler=PimFileTransferService(),
```

Comment:
@hpohekar this overrides the user's selection.

Reply:
Corrected. Thank you.